Tools/ardupilotwaf/cxx_checks.py

```diff
@@ -257,6 +257,9 @@ def check_SFML(cfg, env):
     if not cfg.options.enable_sfml:
         cfg.msg("Checking for SFML graphics:", 'disabled', color='YELLOW')
         return False
+    if cfg.env.DEST_OS == 'darwin':
+        cfg.env.append_value('INCLUDES', ['/usr/local/include'])
```

Comment:
This is rather odd.

Presumably SFML has been installed from a non-packaged source?

If so - should we get some documentation together?

Reply:
The easiest way to install SFML on macos is to use homebrew - that's what gives this location.

BUT it doesn't work in SITL because there is a macos limitation in SFML that prevents you doing any video operations from anywhere but the main thread. So to make this work requires some retooling of the sitl backend.

Reply:
So should these lines be removed as causing noise ATM?

Reply:
Removed